packages/aws-cdk-lib/aws-appconfig/lib/environment.ts

```diff
@@ -561,15 +561,17 @@ export interface IEnvironment extends IResource {
   addExtension(extension: IExtension): void;
 
   /**
-   * Grant the given identity the specified actions
-   * @param grantee the identity to be granted the actions
-   * @param actions the data-access actions
+   * Adds an IAM policy statement associated with this environment to an IAM principal's policy.
+   *
+   * @param grantee the principal (no-op if undefined)
+   * @param actions the set of actions to allow (i.e., 'appconfig:GetLatestConfiguration', 'appconfig:StartConfigurationSession', etc.)
```

Comment:
I think the comma after i.e. can be removed, right? 🤔

Reply:
@daschaa 
I'm adding a comma (,) to align with the descriptions for DynamoDB.

https://github.com/aws/aws-cdk/blob/4b6dc8c650822bcd0231c8890bd94a934a0cd34d/packages/aws-cdk-lib/aws-dynamodb/lib/table-v2-base.ts#L69